docs/release_notes.rst

```diff
@@ -1,16 +1,115 @@
+
+This release of Bodhi has a few changes that are technically backward incompatible in some senses,
+but it was determined that each of these changes are justified without raising Bodhi's major
+version, often due to features not working at all or being unused. Justifications for each are given
+inline.
+
+* dnf and iniparse are now required dependencies for the Python bindings. Justification:
+  Technically, these were needed before for some of the functionality, and the bindings would
+  traceback if that functionality was used without these dependencies being present. With this
+  change, the module will fail to import without them, and they are now formal dependencies.
+* Support for EL 5 has been removed in this release. Justification: EL 5 has become end of life.
+* The pkgtags feature has been removed. Justification: It did not work correctly and enabling it was
+  devastating
+  (`#1634 <https://github.com/fedora-infra/bodhi/issues/1634>`_).
+* Some bindings code that could log into Koji with TLS certificates was removed. Justification: It
+  was unused
+  (`b4474676 <https://github.com/fedora-infra/bodhi/commit/b4474676>`_).
+* Bodhi's short-lived ``ci_gating`` feature has been removed, in favor of the new
+  Greenwave integration feature. Thus, the ``ci.required`` and ``ci.url`` settings no longer
+  function in Bodhi. The ``bodhi-babysit-ci`` utility has also been removed. Justification: The
+  feature was never completed and thus no functionality is lost
+  (`#1733 <https://github.com/fedora-infra/bodhi/pull/1733>`_).
 
-Dependency changes
-------------------
 
-dnf is now a required dependency for the Python bindings. Technically, it was needed before for some
-of the functionality, and it would traceback if that functionality was used without dnf being
-present. With this change, the module will fail to import without dnf, and dnf is a formal
-dependency.
+Features
+^^^^^^^^
+
+* There are new search endpoints in the REST API that perform ilike queries to support case
+  insensitive searching. Bodhi's web interface now uses these endpoints
+  (`#997 <https://github.com/fedora-infra/bodhi/issues/997>`_).
+* It is now possible to search by update alias in the web interface
+  (`#1258 <https://github.com/fedora-infra/bodhi/issues/1258>`_).
+* Exact matches are now sorted first in search results
+  (`#692 <https://github.com/fedora-infra/bodhi/issues/692>`_).
+* The CLI now has a ``--mine`` flag when searching for updates or overrides
+  (`#811 <https://github.com/fedora-infra/bodhi/issues/811>`_,
+  `#1382 <https://github.com/fedora-infra/bodhi/issues/1382>`_).
+* The CLI now has more search parameters when querying overrides
+  (`#1679 <https://github.com/fedora-infra/bodhi/issues/1679>`_).
+* The new case insensitive search is also used when hitting enter in the search box in the web UI
+  (`#870 <https://github.com/fedora-infra/bodhi/issues/870>`_).
+* Bodhi is now able to query Pagure for FAS groups for ACL info
```

Comment:
@ryanlerch Technically Bodhi can work with any Pagure instance. Since Bodhi is used by more than just Fedora, it's probably better to just say we can integrate with Pagure.